webcompat/views.py

```diff
@@ -165,8 +165,9 @@ def create_issue():
         bug_form = get_form(request.headers.get('User-Agent'))
         if g.user:
             get_user_info()
-        if request.args.get('src'):
-            session['src'] = request.args.get('src')
+        for param in ['src', 'label']:
```

Comment:
I should probably leave a comment here, explaining why `src` and `label` are special, and might need to end up in the session cookie.